spec.markdown

```diff
@@ -354,15 +354,16 @@ The following algorithm details these restrictions:
     `cors` or `default`, return `true`.
 3.  Return `false`.
 
-<p class="note">The [response types][response type] are defined by the Fetch
+<div class="note">The [response types][response type] are defined by the Fetch
```

Comment:
Hm, there seem to have been some HTML sanitization on my previous comment :-) Here's another try:

"As a 'p', the '*' below are not rendered as a list; the note needs to be a 'div' for this to work."